### ISSUE TITLE
python scripts should be in UNIX file format

### ISSUE THREAD

**Issue by tjanas** [frederic-mahe/Hardware-Target-Game-Database#675]
When running in Windows, build_pack.py / parse_pack.py scripts are able to run whether they are in DOS or UNIX format.
However, they will not run in Linux if they are in DOS file format; python will produce the following error: `/usr/bin/env: ‘python3\r’: Not a directory`

All of the python scripts should be converted to UNIX format.

**Comment by frederic-mahe** [comment]
I've chosen to use a DOS format to make it easier for Windows users, as it is not clear to me how Windows deals with scripts in Unix format.

Here is what works or doesn't work fo Linux users (like myself):
```sh
python3 parse_pack.py
# works as expected

chmod +x parse_pack.py
./parse_pack.py
# error

dos2unix parse_pack.py
./parse_pack.py
# works as expected
```
So, I'd recommend Linux users to call python first, or to convert the script to Unix if they want to execute the script directly.

**Comment by frederic-mahe** [comment]
Maybe the README file could be modified to carry that info?

**Comment by piyoeu** [comment]
Git allows to configure the line endings, as described here: https://docs.github.com/en/get-started/getting-started-with-git/configuring-git-to-handle-line-endings

That way everyone could use his system's native file format and checkout and commit operations would automaticly convert the line endings into the internal format (which should be UNIX)

**Comment by frederic-mahe** [comment]
brilliant! thanks @piyoeu I didn't know that Git can handle line endings automatically.

Fixed with commit 1b07ca0ee5c0a393c5d5be8f38d4b443bddb7cdd
